STAR-Delegation/CSR-template/test/GOOD_full.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
So I guess we should include this file from the draft.

Reply:
My intent was to make this a test suite for cross validation JSON/CDDL, not to include it in the draft.

The good old example is automatically checked too, so it should be sufficient proof it's not BS I guess.

Reply:
The good old schema still includes two IP addresses, so something's missing (maybe just `"additionalProperties": false`).

Reply:
no, my fault, I didn't do a `make` from top-level or a `make check-example` from the CSR-template dir before submitting...